source/_integrations/atag.markdown

```diff
@@ -0,0 +1,188 @@
+---
+title: Atag
+description: Instructions on how to setup Atag integration.
+logo: atag.png
+ha_category:
+  - Climate
+  - Water heater
+  - Sensor
+ha_release: 0.105
+ha_iot_class: Local Polling
+---
+
+The `Atag` integration allows Home Assistant to connect to [Atag One](https://atag-one.com) thermostats, reporting and setting its status.
+The integration implements the following platforms.
+
+- Climate
+- Water Heater
+- Sensor
+
+## Configuration
+
+The Atag integration can be enabled directly from the config flow UI.
+In a regular setup only the IP address has to be provided (UDP discovery planned for future release).
+
+Alternatively the integration can be setup through configuration.yaml.
+This way additional (non-default) sensors can be enabled. Only the `host` field is required.
```

Comment:
Comma:
```suggestion
This way, additional (non-default) sensors can be enabled. Only the `host` field is required.
```

Furthermore, this sounds weird. Why would a manual configuration via YAML give more sensors?